docs/whitepaper.tex

```diff
@@ -157,80 +157,83 @@ \section{Implementation details}
   When providing liquidity, $x \rightarrow x'$ and $y \rightarrow y'$ has the
   effect that $k \rightarrow k'$. The number of liquidity tokens the LP
   receives, $\Delta L$, then follows the rule $\Delta L / L = \Delta k / k$.
-}, we need a liquidity token to exist, with a minting policy linked to a
-specific pool.
-Secondly, as in the Cardano Application Template, we want each pool be
-identified by an NFT minted through a specific minting policy. This allows
-anyone to verify that a UTXO at the pool script address is valid, as well as to
-trace each pool.
-Finally, our protocol contains a number of cyclic dependencies between the
-different scripts. In order to resolve this, we follow the Config UTXO pattern.
+}, we need a liquidity token linked to the respective pool to exist. 
+This implies that the liquidity token needs to be minted/ burnt using a minting policy linked to a specific pool.
+Secondly, as in the Cardano Application Template, we want the liquidity pools to be
+identifiable, traceable and verifiable (valid or not).
+We achieve this by minting an identifier NFT which requires its own minting policy.
+Lastly, our protocol consists of a number of scripts whose functionalities are dependent
+on each other, where some of those functional dependencies are cyclic. We model/ lock these
+functional dependencies using the \textit{Config UTxO} pattern.
```

Comment:
It may be useful to provide a footer link to somewhere this pattern is documented by the larger community.

Reply:
Wait, is this something we proposed or was this proposed by the community? If the latter case is true then our wording was misleading. Reading the paper i thought it was something we came up with

Reply:
Oh, I don't know where the idea came from but I'm fairly sure it wasn't any of us.

Reply:
I removed the usage of propose